newIDE/app/src/SceneEditor/index.js

```diff
@@ -2196,6 +2254,7 @@ export default class SceneEditor extends React.Component<Props, State> {
                     hotReloadPreviewButtonProps={
                       this.props.hotReloadPreviewButtonProps
                     }
+                    isListLocked={true}
```

Comment:
Can you confirm this is fine and that we don't need additional changes to unblock some authorised changes to the variables?

Reply:
The instance variable editor allows to reset back an instance variable to the default value. I guess it allows to remove undeclared instance variables.

There is probably something to do to help users understand why the "add" button is disabled and how they can add an object variable, but I guess this is already an improvement as it now forbids to add an undeclared instance variable which could be miss-understood as adding an object variable and sound like a bug when the events give no auto-completion for it.